open-api/rest-catalog-open-api.yaml

```diff
@@ -1482,6 +1490,38 @@ components:
       explode: false
       example: "vended-credentials,remote-signing"
 
+    page-token:
+      name: pageToken
+      in: query
+      description:
+        A unique token which allows clients to make use of pagination by signaling to the service that they would
```

Comment:
nit: I would say an `opaque` as opposed `unique` token as there is no requirement that the token is unique.